extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -33,6 +33,9 @@ define('ext.wikia.adEngine.slotTweaker', [
 		if (slot) {
```

Comment:
What do you think about merging the conditions in terms of readability?

``````
if (a && b) {
   // do something for a
   // do something for b
} else if (a && !b) {
   //do something for a
}
```
instead of:
```
if (a) {
   // do something for a
   if (b) {
      // do something for b
   }
}
```